### DISCUSSION TITLE
Stand Up Thursday April 20th

### DISCUSSION THREAD

**Discussion by BrandonMagana** [Bugs-io/TC3001B#63] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Continue with form control on signup
 - Advanced Web class' activity
 
**What are your priorities for the next 24 hours?**:
 - Get as close as I can to finish the signup and login
 
**Blockers**:
 - Freaking Carbon Design does not have "required" implemented nor strength meter, so it's taking a little more time that expected

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Rebased hex-architecture-rebase branch
 - Worked with @BrandonMagana to develop first version of signup/login

**What are your priorities for the next 24 hours?**:
 - Finish signup/login endpoints
 - Work on authentication middleware for the rest of the endpoints
 
**Blockers**:
 - Please check my PR

**Shoutouts**:
 - @CristianCazares for his work on the web stuff

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Designed Database basic structure with @CarlosCRG19 
 -  Worked with @CarlosCRG19 on login feature 
  
**What are your priorities for the next 24 hours?**:
 - Finish basic login implementation using local db 
 - Review pending prs
 
**Blockers**:
 - Web homework
 
**Shoutouts**:
 - @CarlosCRG19 for his backed architecture design
 - @CristianCazares for working on web homework
